pocket-notes/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pocket Notes</title>
+
+    <!--Link of external css files-->
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+
+    <!--Link of font awesome cdn-->
+    <script
+      src="https://kit.fontawesome.com/04202a2ef3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!--Start of wrapper-->
+    <div class="wrapper">
+      <!--Start of header-->
+      <header class="header">
+        <!--Start of container-->
+        <div class="container">
+          <h3>POCKET NOTES</h3>
+          <!--Start of view selection -->
+          <div class="view-selection">
+            <p>Default View</p>
+            <!--Start of select -->
+            <select name="view-selection-dropdown" id="viewBySelection">
```

Comment:
Can this be populated from JS. Have constants variable to store the different format labels and values.